src/UI/Root/Components/Header/EnvSelector/EnvironmentSelector.test.tsx

```diff
@@ -131,3 +135,62 @@ test("GIVEN EnvironmentSelector and environments with identical names WHEN user
 
   expect(selectedEnv).toEqual(envB.id);
 });
+
+const setup = (config: KeycloakAuthConfig | LocalConfig | undefined) => {
```

Comment:
Why not declare this at the top of the file and reuse the setup for the other tests too?

Reply:
changed